
try{
    console.log(varInexistent)
}catch(err){
    console.log('varInexistente NÃ£o Existe')
    console.log(err)//err mostra o log do erro
}